IO/NativeSignalHandlers/WindowsNTHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Runtime.InteropServices;
+
+namespace LocalAdmin.V2.IO.NativeSignalHandlers
+{
+    /// <summary>
+    ///     Native signal processing on Windows NT systems.
+    /// </summary>
+    internal sealed class WindowsNTHandler : INativeSignalHandler
+    {
+        public static readonly WindowsNTHandler Handler;
+
+        static WindowsNTHandler()
+        {
+            Handler = new WindowsNTHandler();
+        }
+
+        private WindowsNTHandler() { }
```

Comment:
This is kindof useless

Reply:
Yes, you’re right, it'll be deleted, I don’t think that someone will create a class instead of using its static `Handler`.

Reply:
Resolved in https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/ec5913123e28886a24af00f4fc0549d7e5501b83